src/databaseQueries/admin.js

```
@@ -32,6 +32,37 @@ module.exports = class Admin {
+					{ [`groups.${entityType}`]: entityId },
+					updateQuery
+				)
+				return resolve(result)
+			} catch (error) {
+				return reject({
+					status: error.status || HTTP_STATUS_CODE.bad_request.status,
+					message: error.message || HTTP_STATUS_CODE.bad_request.message,
+					errorObject: error,
+				})
+			}
+		})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)